test/test_rosbag/bag_migration_tests/scripts/generate_data

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Function that builds a message generation:
+build_generation()
+{
+    catkin build --no-deps --no-notify --no-status --no-summary --cmake-args -DMSG_DIRECTORY="msg_$1" -- test_rosbag \
+        >/dev/null
```

Comment:
We probably don't want a `catkin_tools` dependency here. Given the simplicity of what's being done, consider just using `cmake; make` directly?